site/source/docs/tools_reference/settings_reference.rst

```diff
@@ -3355,3 +3355,15 @@ It's currently only available under a flag in certain browsers,
 so we disable it by default to save on code size.
 
 Default value: false
+
+.. _cross_origin_workers:
+
+CROSS_ORIGIN_WORKERS
```

Comment:
Should we just name it `CROSS_ORIGIN` for future-proofing? Do you think we would ever need this workaround for other Emscripten resources? I assume JS and Wasm already work across origins with our JS as-is, but idk about all the graphics / fonts / whatever stuff Emscripten loads in some libraries.

Reply:
OK, sounds reasonable.